projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl

```diff
@@ -133,54 +133,6 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{- end -}}
 {{- end -}}
 
-{{/*
```

Comment:
May you link the PR(s), and say a few words on 1) what exactly got deprecated 2) in favour of what new config
added to the description?